infra/modules/aws-oauth-refresh-strategy/main.tf

```diff
@@ -0,0 +1,47 @@
+resource "aws_ssm_parameter" "long_refreshTokens_client_id" {
```

Comment:
A new module to encapsulate connectors based on refresh tokens. We need clientId, clientSecret and refreshToken